src/cmd/user.rs

```diff
@@ -1,9 +1,12 @@
+use crate::legacy::{prepare_args, prepare_flags};
+use anyhow::Result;
 use clap::{Args, Subcommand};
+use std::{collections::HashMap, process::Command};
 
 #[derive(Args)]
 pub struct User {
     #[clap(subcommand)]
-    command: UserCommands,
+    pub command: UserCommands,
```

Comment:
Needed to make this public for the `whoami` command alias here 

https://github.com/fission-codes/fission-cli/blob/4f48f7cd56f972129f89af87ce34507285d4d33f/src/main.rs#L85

Would be open to alternative approaches!